### PR TITLE
Add RS automation requirements

### PR DESCRIPTION
**Pull request by HarshitSohaney**
## Description

In order to automate uploading lists to RS, we need specific .ini files and a Dockerfiler to run `lists2safebrowsing.py` automatically.

## Requirements

- [x] Create separate `.ini` files for different environments 
- [x] Add a Dockerfile for automatically running `lists2safebrowsing.py` in the environment
- [x] Add ability in code base to read different `.ini` files based on environment variables

#### Note 1

I was unable to test these changes completely since I don't have the environment locally running. I would need some help to test this.

#### Note 2

We will have to update readme once this gets merged in to reflect what these files do and how they are used.

### CODE REVIEW

File: lists2safebrowsing.py [hunk 1]
```diff
@@ -663,10 +663,17 @@ def start_versioning(config, chunknum, shavar_prod_lists_branches):
...
 
 def main():
+    execution_environment = os.getenv("EXECUTION_ENVIRONMENT", "JENKINS")
+    environment = os.getenv("ENVIRONMENT", "stage")
+
     config = configparser.ConfigParser()
     filename = config.read(["shavar_list_creation.ini"])
+
+    if execution_environment == "GKE":
+        filename = config.read([f"{environment}.ini"])
```
Comment (artines1) [inline lists2safebrowsing.py:673]: Could we avoid reading the .ini file twice? If the `ENVIRONMENT` is set and the environment is GKE, we don't actually need to read from the `shavar_list_creation.ini` file.

Comment (HarshitSohaney) [reply lists2safebrowsing.py:672]: Made it into an if else statement!

File: lists2safebrowsing.py [hunk 2]
```diff
@@ -663,10 +663,23 @@ def start_versioning(config, chunknum, shavar_prod_lists_branches):
...
 
 def main():
+    execution_environment = os.getenv("EXECUTION_ENVIRONMENT", "JENKINS")
+    environment = os.getenv("ENVIRONMENT", "stage")
+
     config = configparser.ConfigParser()
-    filename = config.read(["shavar_list_creation.ini"])
+
+    if execution_environment == "GKE":
+        filename = config.read([f"{environment}.ini"])
```
Comment (artines1) [inline lists2safebrowsing.py:672]:
```suggestion
        filename = config.read([f"rs_{environment}.ini"])
```
So we can match the file name you use in the repo.

**Review by artines1 — CHANGES_REQUESTED** [review]

File: lists2safebrowsing.py (hunk 2 above)
Comment (HarshitSohaney) [reply lists2safebrowsing.py]: Fixed!

**Review by artines1 — APPROVED** [review]

**Comment by artines1** [comment]
Let 's wait for ahoneiser to take a look at the Dockerfile before merge.

**Review by artines1 — CHANGES_REQUESTED** [review]
The .ini file also gets read in the publish2cloud.py, we also need to change it.

**Review by artines1 — APPROVED** [review]
LGTM

**Comment by HarshitSohaney** [comment]
I found an error with the parsing, just fixed it (I forgot to add os.environ to the `ConfigParser()`

**Review by ahoneiser — APPROVED** [review]
lgtm

**Comment by artines1** [comment]
Fix #187

**Pull request merged**